### PR TITLE
Create works and about sections

### PR DESCRIPTION
**Pull request by sameedmohsin28**
Created the works and about sections of the portfolio: mobile version. It took me four days to complete this task. I have styled these sections using CSS Grids and CSS Flexbox as mentioned in the task description.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -65,5 +62,218 @@ <h1 class = "top-heading"> Hi there! I'm Sameed</h1>
...
+
+    </section>
+
+    <section class="about-section">
+
+        <article class="about-myself">
+            <h1 class = "about-heading">About<br>Myself
+            </h1>
+            <p class="about-description">
+                Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
```
Comment (frank1738) [inline index.html:192]: - [ ] Kindly adjust the padding around your `about me ` section to match the original design and also adjust the border-radius on the top right corner

image

### Checkout the original design

image

File: index.html [hunk 2]
```diff
@@ -65,5 +62,218 @@ <h1 class = "top-heading"> Hi there! I'm Sameed</h1>
...
+                        See Project
+                    </span>
+                </div>
+            </div>
+        </article>
+
+        <article class="first-project">
+            <img src="./Images/4 multi-post stories.png" alt="" class="work-image">
+            <div class="details-container">
+                <div class="heading-and-highlights">
```
Comment (frank1738) [inline index.html:158]: - [ ] Kindly note in the original design, the project cards have a background of `ffffff` while the whole section has a background color of `e5e5e5`. I would suggest that you bring out that contrast.

image

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @sameedmohsin28 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights

✅ Correct use of Github workflow
✅ No linter error

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (sameedmohsin28) [reply index.html:158]: when I set e5e5e5 as the background color, it changes the colour of the first page (intro page as well). When I set its background colour as white, the ellipses and other shape in the beckground get hidden behind the white color. How do I fix that?

**Review by Tadesse-Alemayehu — CHANGES_REQUESTED** [review]
### Hi @sameedmohsin28 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)

## Highlights
 - [ ]  Good job making the changes requested by the previous code reviewer. ✅
 - [ ]  Correct GitHub Flow. ✅
 - [ ]  Good commit history. ✅
 - [ ]  Good PR title/description. ✅
 - [ ]  All linters passed. ✅
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Tadesse-Alemayehu — APPROVED** [review]
### Hi @sameedmohsin28 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)

## Highlights
 - [x]  Good job making the changes requested by the previous code reviewer. ✅
 - [ ]  Correct GitHub Flow. ✅
 - [ ]  Good commit history. ✅
 - [ ]  Good PR title/description. ✅
 - [ ]  All linters passed. ✅
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**